qiskit_dynamics/solvers/perturbative_solvers/expansion_model.py

```diff
@@ -60,36 +60,40 @@ def __init__(
 
         Args:
             operators: List of constant operators specifying the operators with signal coefficients.
-            rotating_frame: Rotating frame to setup the solver in.
-                            Must be Hermitian or anti-Hermitian.
+            rotating_frame: Rotating frame to setup the solver in. Must be Hermitian or
+                anti-Hermitian.
             dt: Fixed step size to compile to.
             carrier_freqs: Carrier frequencies of the signals in the generator decomposition.
             chebyshev_orders: Approximation degrees for each signal over the interval [0, dt].
             expansion_method: Either 'dyson' or 'magnus'.
-            expansion_order: Order of perturbation terms to compute up to. Specifying this
-                                argument results in computation of all terms up to the given order.
-                                Can be used in conjunction with ``expansion_terms``.
+            expansion_order: Order of perturbation terms to compute up to. Specifying this argument
+                results in computation of all terms up to the given order. Can be used in
+                conjunction with ``expansion_terms``.
             expansion_labels: Specific perturbation terms to compute. If both ``expansion_order``
-                                and ``expansion_terms`` are specified, then all terms up to
-                                ``expansion_order`` are computed, along with the additional terms
-                                specified in ``expansion_terms``. Labels are specified either as
-                                ``Multiset`` or as valid arguments to the ``Multiset`` constructor.
-                                This function further requires that ``Multiset``\s consist only of
-                                non-negative integers.
+                and ``expansion_terms`` are specified, then all terms up to ``expansion_order`` are
+                computed, along with the additional terms specified in ``expansion_terms``. Labels
+                are specified either as ``Multiset`` or as valid arguments to the ``Multiset``
+                constructor. This function further requires that ``Multiset``\s consist only of
+                non-negative integers.
             integration_method: ODE solver method to use when computing perturbation terms.
             include_imag: List of bools determining whether to keep imaginary components in
-                          the signal approximation. Defaults to True for all signals.
+                the signal approximation. Defaults to True for all signals.
             kwargs: Additional arguments to pass to the solver when computing perturbation terms.
 
         Raises:
-            QiskitError: if invalid expansion_method passed.
+            QiskitError: If invalid expansion_method passed.
```

Comment:
(did not check if this needs two lines)
```suggestion
            QiskitError: If invalid ``expansion_method`` passed, or if arguments have incompatible lengths.
```